ui/js/plugins/fasagreement/fasagreement.js

```diff
@@ -1,150 +1,136 @@
 //
 // FreeIPA plugin for Fedora Account System
-// Copyright (C) 2019  Christian Heimes <cheimes@redhat.com>
+// Copyright (C) 2019  FreeIPA FAS Contributors
 // See COPYING for license
 //
 
 define([
-        'freeipa/phases',
```

Comment:
Please disable your automatic code formatter and do the copyright change again in this file. Most of those changes are unrelated to this PR.

Reply:
My bad! I'll do that now. 

Thank you